docs/reference/technologies/client/web/index.mdx

```diff
@@ -230,6 +237,24 @@ const domainScopedClient = OpenFeature.getClient("my-domain");
 Domains can be defined on a provider during registration.
 For more details, please refer to the [providers](#providers) section.
 
+#### Manage evaluation context for domains
```

Comment:
Wait to merge until this feature has been released.

https://github.com/open-feature/js-sdk/pull/927